packages/lit-dev-content/site/_includes/default.html

```diff
@@ -16,7 +16,8 @@
     {% inlinejs "global/initialize-typescript-attribute.js" %}
     {% inlinejs "global/mobile-drawer.js" %}
 
-    {% if showOldDocsBanner %}
+    {% if selectedVersion !== latestVersion %}
+      <link rel="canonical" href="{{ versions[latestVersion].path }}/{{ versionLinks[latestVersion] }}">
```

Comment:
should there be another check that there is a versions[latestVersion] just in case?

Reply:
Hmm, not sure of an easy way to do that -- I dont see an "assert" or anything in nunjucks. I guess we could have a custom directive. But I sort of feel like there are dozens of similar things in our templates that are similarly brittle...